docs/src/docs/core/Grid.mdx

```diff
@@ -31,7 +31,7 @@ Use xs, sm, md, lg, xl values to set spacing from `theme.spacing` or number to s
 
 ## Grow
 
-Set `grow` prop on Grid component to force last row take 100% of container width:
+Set `grow` prop on Grid component to force all rows to take 100% of container width:
```

Comment:
Unrelated - this documentation is incorrect. `grow` affects all rows, not just the last.